src/components/views/elements/AppTile.js

```diff
@@ -340,7 +350,13 @@ export default class AppTile extends React.Component {
                 <Spinner message={_t("Loading...")} />
             </div>
         );
-        if (!this.state.hasPermissionToLoad) {
+        if (this._sgWidget === null) {
+            appTileBody = (
+                <div className={appTileBodyClass} style={appTileBodyStyles}>
+                    <AppWarning errorMsg="Error loading Widget" />
```

Comment:
Translations?

Reply:
oh yeah - I guess I'll change the mixed content one I copied it from too